site/components/TransactionsModal.js

```diff
@@ -0,0 +1,146 @@
+import useTranslation from 'next-translate/useTranslation'
+
+import { EtherscanLink } from './EtherscanLink'
+import JustifiedBetweenRow from './JustifiedBetweenRow'
+import Modal from './Modal'
+import SvgContainer from './svg/SvgContainer'
+import { useNumberFormat } from '../hooks/useNumberFormat'
+
+const TransactionModalRow = function ({ text, value, tipLink = '' }) {
+  return (
+    <JustifiedBetweenRow
+      keyComponent={
+        <p
+          className={`text-gray-350 text-sm ${
+            tipLink ? 'flex justify-bewtween cursor-pointer' : ''
+          }`}
+          onClick={() => tipLink && window.open(tipLink, '_blank')}
+        >
+          {text}
+          {tipLink && <SvgContainer className="ml-1" name="questionmark" />}
+        </p>
+      }
```

Comment:
Should these be under an `anchor` tag? sounds more semantic

Reply:
It's supposed to be a link only if there is a tip. But certainly is a bit confusing.

Reply:
Changed in b73e76b.